docs/components/course/course-nav/course-nav-chapter.md

```diff
@@ -0,0 +1,72 @@
+CourseNavChapter
+===============
+This component is used to display the follow edit view of a course nav.
+## Structure
+
+### Roots
+- [`CourseNavigationEditor`](...)
+
+### Dependencies
+* `Draggable`from`vuedraggable`
+* `chapterFollowSet`from`@/mixins/general/course-chapters`
+* `deepCopy`from`@/mixins/general/helpers`
+* `v4 as uuidv4`from`uuid`
+
+### Components
+- [`CourseNavItem`](...)
+- [`CourseNavPropertyEdit`](...)
+
+### Mixins
+* `courseNav`
+* `locale`
+
+Component Functionality
+---------
+### Props
+- `chapterNameDuplicate`: type: Boolean, default set to false
+- `chapterName`: type: String, default set to `chapterName`
+- `chapter`: type: Object, required: true
+- `courseEnd`: type: String, required: true
```

Comment:
`required` would be enough; props either need to be required or have a default set

Reply:
that means only required without : true?

Reply:
yes, `- `chapter`: type: Object, required` would suffice